CMakeLists.txt

```diff
@@ -72,11 +72,25 @@ else()
     add_subdirectory(.cppan)
 endif()
 
-hunter_add_package(OpenCL)
-find_package(OpenCL CONFIG REQUIRED)
-
 option(BUILD_TRAINING_TOOLS "Build training tools" ON)
 option(Tesseract_BUILD_EXE "Build tesseract executable" OFF)
+option(Tesseract_USE_OPENCL "Build tesseract with OpenCL support" ON)
+
+if(Tesseract_USE_OPENCL)
+    if (MSVC OR MSYS OR (UNIX AND NOT APPLE AND NOT ANDROID))
+        # Supported platforms for OpenCL ICD in Hunter
+        hunter_add_package(OpenCL)
+        find_package(OpenCL CONFIG REQUIRED)
+    else()
+        # Possible support via NDK
+        message("Attempting to use inbuilt OpenCL development files")
+        find_package(OpenCL QUIET)
```

Comment:
Ah so it should just be if(APPLE OR ANDROID) ... else() .. as you said above.

Ok. Thanks.